bubbletea/bubbletea.go

```diff
@@ -56,7 +56,15 @@ type BubbleTea struct {
 	currentModel Model
 
 	// models are the models of the `BubbleTea` component.
+	// `BubbleTea` keeps track of unique model, tracked by type.
```

Comment:
Since this code comment do not apply anymore, I went ahead and removed it via: 3c73bb3.